common/frame_manager.go

```diff
@@ -390,10 +390,16 @@ func (m *FrameManager) frameRequestedNavigation(frameID cdp.FrameID, url string,
 	return nil
 }
 
-func (m *FrameManager) getFrameByID(id cdp.FrameID) *Frame {
+// getFrameByID will attempt to find a frame with the given id. If one is found
+// the frame and true will be returned. If one isn't found nil and false will
+// be returned.
```

Comment:
What about using the present tense as the rest of the code?

```suggestion
// getFrameByID finds a frame with id. If found, it returns the frame and true,
// otherwise, it returns nil and false.
```